doc/plugins/eu.esdihumboldt.hale.doc.user/html/reference/import/geopackage.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>GeoPackage Data Import</title>
+</head>
+<body>
+	<h1>GeoPackage Data Import</h1>
+	<p>
+		<i>hale studio</i> supports the import of GeoPackages. 
```

Comment:
Maybe at least add the hint that usually the Geopackage first needs to be imported as schema, then as data?